src/shared/bus-log-control-api.h

```diff
@@ -2,7 +2,12 @@
 
 #include "sd-bus.h"
 
-int bus_log_control_api_register(sd_bus *bus);
+#include "bus-util.h"
+
+extern const BusObjectImplementation log_control_object;
+static inline int bus_log_control_api_register(sd_bus *bus) {
+        return bus_add_implementation(bus, &log_control_object, NULL);
+}
```

Comment:
why export the variable? why not hide it away in bus-log-control-api.c and make bus_log_control_api_register() a function like any other that is not inline?

Reply:
It's used directly as an argument to service_parse_argv().